pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/Flake8Task.java

```diff
@@ -32,8 +34,37 @@
 public class Flake8Task extends AbstractPythonMainSourceDefaultTask {
 
     private static final Logger log = Logging.getLogger(Flake8Task.class);
+    // Track whether the current run is excluding the new rules
+    private Boolean ignoringNewRules = false;
+    private String firstRunOutput = null;
+
+    /* Set of Flake8 rules that were added in version bumps. This set of rules, and this comment should be updated
```

Comment:
Let's use the commenting style for long (paragraph) comments that's same as in the rest of the code:
```
/*
 * Set of flake8 rules ... version updates.
 * ...
 */